### ISSUE TITLE
OData path parameter - '$' dollar sign

### ISSUE THREAD

**Issue by nastassiar** [swagger-api/swagger-ui#1469]
Sorry If this issue is in the wrong place but I'm having an issue very similar to:
https://github.com/swagger-api/swagger-editor/issues/308

I am using Swashbuckle.Core.5.2.1.

I am trying to add Odata Parameters to my Get queries
```
operation.parameters.Add(new Parameter
{
     name = "$top",
     required = false,
     type = "string",
     @in = "query",
     description = "Maximum number of results to return"
});
```
I am expecting the resulting url to look like **/apps?$top=&$skip=1**

But instead it is coming out as **/apps?%24top=2&%24skip=1**

**Comment by nastassiar** [comment]
Sorry I think I was just confused

**Comment by steve-o** [comment]
This is still a continuing issue with Swagger as of June 2nd 2016, it is resolved in the editor (2.10.1) per case #308  but not the UI (2.1.4) and so highly inconvenient when defining OData queries with pagination or change tracking, for example:

```
"parameters": [
    {
        "name": "Prefer",
        "in": "header",
        "required": false,
        "type": "string",
        "description": "For paging request set to \"odata.maxpagesize=10\", where 10 is the maximum number of files to return in each response.  For change tracking set t
o \"odata.track-changes; odata.maxpagesize=10\"."
    },
    {
        "name": "$skiptoken",
        "in": "query",
        "required": false,
        "type": "string",
        "description": "For paging request, a token to refer to a previous request."
    },
    {
        "name": "$deltatoken",
        "in": "query",
        "required": false,
        "type": "string",
        "description": "For change tracking, a token to refer to a previous request."
    }
]
```
